src/Simplex_tree/test/simplex_tree_make_filtration_non_decreasing_unit_test.cpp

```diff
@@ -37,7 +37,7 @@ struct Simplex_tree_options_stable_simplex_handles {
 
 typedef boost::mpl::list<Simplex_tree<>,
                          Simplex_tree<Simplex_tree_options_fast_persistence>,
-                         Simplex_tree<Simplex_tree_options_fast_cofaces>,
+                         Simplex_tree<Simplex_tree_options_full_featured>,
```

Comment:
Just checking: do we have at least one test that has options link=true, stable=false?